_storage/amazon-s3-deploy.md

```diff
@@ -0,0 +1,466 @@
+
+```
+$ which watch
+/usr/local/bin/watch
+```
+
+If `watch` is not located, install it on your workstation.
+
+## Task 1: Launch an EC2 instance
+
+1. Visit the <a href="https://aws.amazon.com/free/" target="\_blank">AWS Free Tier page</a> and choose **Sign in to the Console**.
+
+   ![](/storage/images/aws-console.png)
+
+2. Make sure your region is set to `us-west-2` which is **Oregon**.
+
+   ![](/storage/images/us-west-2.png)
+
+3. Choose **Launch a virtual machine**.
+
+   The system opens the EC2 dashboard.
+
+4. Enter `Gaia` in the search bar.
+
+   The system finds AMIs in the Marketplace and the Community.
+
+5. Choose **Community AMIs**.
+
+   The system displays the available Gaia hub images.
+
+   ![](/storage/images/gaia-community.png)
+
+6. Select the **Blockstack.Gaia.Hub_1549643256 - ami-04793ff666176c1f3** image.
+
+   The system displays **Step 2: Choose an Instance Type** page.
+
+    ![](/storage/images/tier-2-image.png)
+
+7. Select **t2.micro** and choose **Configure Instance Details**.
+
+   1. Select a VPC.
+
+       A default VPC is created with a free tier account. You can use this
+       default VPC. Or you can choose another VPC. If you choose another VPC,
+       ensure the `Subnet` value is set to a subnet reachable by a public IP.
+
+       {% include important.html content="If you're using a private subnet, you
+       should attach an elastic IP (EIP) to the VM. This EIP allows you to
+       reboot the instance without worrying whether the address will reset. To
+       attach an IP, <strong>press allocate new address</strong> and follow the
+       instructions to attach the EIP to your new EC2 instance." %}
+
+   2. Set **Protect against accidental termination**.
+
+      If you terminate a Gaia instance, you lose all the data associated with it. Protection adds an extra step to terminating your Gaia instance.
+
+   3. Display the **Advanced Details**.
+
+      At this point, you are going to configure environment variables for your instance.  
+
+   4. Paste the following into the **Advanced Details**.
+
+      ```json
+      {
+        "ignition": { "version": "2.2.0" },
+        "storage": {
+          "files": [{
+            "filesystem": "root",
+            "path": "/etc/environment",
+            "mode": 420,
+            "contents": {
+              "source": "data:application/octet-stream,API_KEY%3D<KEYPHRASE>%0ADOMAIN%3D<DOMAIN_NAME_VALUE>%0ASTAGING%3D<STAGING_VALUE>"
+            }
+          }]
+        }
+      }
+      ```
+
+      A value of `0` to requests a valid SSL certificate. At this point, you'll
+      request a staging certificate (`0`).
+
+   5. Replace `<KEYPHRSE>` with a key phrase.
+
+         For example, `hubbadubbado` is a fun key phrase.
+
+   6. Replace `<DOMAIN_NAME_VALUE>` with your domain name.
+
+      For example, `maryhub.ml` is my domain name.
+
+   7. Replace the `<STAGING_VALUE>` with the number `0`.
```

Comment:
If we're asking them to replace this with a constant value of `0`, should we just place that directly into the blob above?

Reply:
I provide an example to show them the result sans the angle brackets.  Most people understand the role of the angle brackets but those who don't can check character by character.  I'll drop in a bit about (be sure not to leave the brackets)

Reply:
I mean to ask, should we just place the staging value as a hardcoded `0` value in the pastable blog instead of including it as a variable? It seems like something that only advanced users would want to change, and they'd know to spot and modify it themselves?

Reply:
Everyone would want to change it to `1` I think